playbooks/vars/defaults.yaml

```diff
@@ -20,6 +20,10 @@ control_plane_ip: 192.168.24.1
 # By default we use the default IP of the host
 public_api: "{{ network_info.public_ipv4.address }}"
 
+# For advanced network configurations (e.g. SR-IOV), network_config can be overriden
+# The format is for os-net-config, check dev-install_net_config.yaml.j2 template.
+# network_config
+
```

Comment:
what if instead of completely removing the network_config, you add to it. 
reason being that then the user does not need to worry about adding the dummy interface or what not. 
Or maybe copy and paste the br-ex and dummy interface definitions to here as an example to be added for a full functioning network_config.

Reply:
There are multiple ways to solve this problem:

- we can add configs to the default but then someone will ask to remove stuffs from the default
- we can completely override the default
- ...

At the end it ends up a bit complex in my opinion, and needs to be maintained.
I think that if you need a network config that isn't the default, you're advanced enough to provide the whole config.

I'm happy to implement whatever solution, but I think we need to keep it stupid and simple and not try to cover all cases.